demos/link-prediction/random-walks/cora-lp-demo.ipynb

```diff
@@ -4,18 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p><code>l1</code>  happened to win this time, so that's why the plot is flipped, but I think it still looks as good</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>